.circleci/config.yml

```diff
@@ -86,6 +96,7 @@ jobs:
       - run:
           name: Setup Test Runner
           command: |
+            # Switch to master branch later
```

Comment:
Do we still need switching branch as long as we are comparing the artifacts now?